data/io.elementary.terminal.gschema.xml

```diff
@@ -43,9 +43,15 @@
         <description>The index of the tab from the saved tabs that is currently focused.</description>
     </key>
     <key name="zoom" type="d">
+    <range min="0.25" max="4.0" />
```

Comment:
What effect do these values have?

I tested in Terminal with 3 tabs: 25% (min), 100% (default) and 400% (max)

These were the values in the settings: `['0,2', '1,0', '4,0']`

Reply:
These values prevent manual entering of out of range default zoom values into the settings using dconf-editor or otherwise.  They are not essential as the code clamps the values it reads in from the settings anyway, but they inform the user.

Reply:
Yes, but do the values make sense if I can bypass them via the UI?

`['0,2', '1,0', '4,0']`

From my point of view it would then have to be changed as follows:

```suggestion
    <range min="0.2" max="4.0" />
```